ConsoleCalculator.edvaudin/ConsoleCalculator.edvaudin.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
Hey there! Glad to have your back! Does your project build? I'm getting this: 
![image](https://user-images.githubusercontent.com/108711476/212571270-f611c36e-e3d2-457f-9254-40cce4dbc5f2.png)